ocp_resources/utils.py

```diff
@@ -26,76 +31,178 @@ class TimeoutSampler:
     Yielding the output allows you to handle every value as you wish.
 
     Feel free to set the instance variables.
+
+    exceptions_dict should be in the following format:
+    {
+        exception0: [exception0_msg0],
+        exception1: [
+            exception1_msg0,
+            exception1_msg1
+        ],
+        exception2: []
+    }
+
+    Args:
+        wait_timeout (int): Time in seconds to wait for func to return a value equating to True
+        sleep (int): Time in seconds between calls to func
+        func (function): to be wrapped by TimeoutSampler
+        exceptions (tuple): Deprecated: Tuple containing all retry exceptions to pass to TimeoutSampler
+        exceptions_msg (str): Deprecated: String to match exception against
```

Comment:
Adding my comment here so it's not lost in the thread; `exceptions_msg` should be removed (the same way un-used cod is removed)

Reply:
https://github.com/RedHatQE/openshift-python-wrapper/pull/24#discussion_r681660501